### DISCUSSION TITLE
Possible to change the state of door-sensor?

### DISCUSSION THREAD

**Discussion by olskar** [thanksmister/android-mqtt-alarm-panel#170] [Q&A] [answered]
Hello!

I have a door sensor publishing its state to mqtt, and it shows up in alarm-panel as on/off.

Is it possible to change this to Open/Closed  @thanksmister  ? :)

**Comment by thanksmister** [comment, marked as answer]
That is something that you can make a new sensor in Home Assistant to reverse or change the states.
